_component_design/tooltip.md

```diff
@@ -17,35 +17,22 @@ helix-ui-javascript: true
 
 <section class="static-section" markdown="1">
 
-## Introduction
-
 <div class="hxRow"  markdown="1">
 
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
-
-### When to use
-
-Use a tooltip when you need to provide in-context help information. (for example, when describing what will happen when a workflow is initiated). A tooltip can also be used to reveal hidden information, provide extra or supplemental information to users, clarify uses when features could be misinterpreted, or explain an item on a page when it does not have a text label.
-
-### When not to use
-
-Do not use a tooltip if users must repeatedly refer to a tooltip to understand an option or to complete a workflow. We recommend you do not use a tooltip when users needs to interact with the tip's content, or when users integrate with the site on a mobile device.
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
 
-### Best practice
+## When to use
 
-- Tooltips are perfect for short sentences and sentence fragments that assist the user in their understanding.
-- The ideal tooltip is not obvious, nor does it repeat content that is already visible on the site.
-- Place a tooltip in a subtle location, and ensure it does not interfere with the user or obscure any objects of interest. - Use the tooltip on icon images, text, and table rows.
-- The maximum width for the tooltip is 400px.
-- Ensure there are no interactive controls inside the tooltip.
+Use a tooltip in the following scenarios.
 
-{% endcolumn %}
+- You want to provide in-context help information to the user. For example, use a tooltip to describe what happens when a user initiates a workflow.
+- To reveal hidden information, or provide extra or supplemental information to users.
+- To clarify uses when features could be misinterpreted, or explain an item on a page when it does not have a text label.
 
-{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+Do not use a tooltip in the following scenarios.
```

Comment:
Just a suggestion that Catherine might change back in editorial, but I would bold or italicize the word *not*... Feel free to ignore this one completely. Not sure if we've done the same on other patterns.